metrics/task.go

```diff
@@ -82,7 +83,10 @@ func (m *metricsTask) Execute() error {
 
 				return nil
 			})
-			if err := m.Subsystem.StatsDClient().Timing(queueLatencyMetricName, timeElapsed, m.Subsystem.Options.Tags, 1); err != nil {
+			if err := m.Subsystem.StatsDClient().Gauge(queueLatencyMetricName, float64(timeElapsed.Milliseconds()), m.Subsystem.Options.Tags, 1); err != nil {
```

Comment:
I may be naive about go, but why does this time number have to be cast to a float64?  Is it because the stats library expects a float for a gauge but the time is an integer?  Worth a comment if it's more complicated than that, but I'm guessing that's it.

Reply:
Yeah it's expecting a float64